src/core/ext/transport/chttp2/transport/flow_control.cc

```diff
@@ -374,7 +374,7 @@ FlowControlAction TransportFlowControl::PeriodicUpdate() {
     // get bandwidth estimate and update max_frame accordingly.
     double bw_dbl = bdp_estimator_.EstimateBandwidth();
     // we target the max of BDP or bandwidth in microseconds.
-    int32_t frame_size = static_cast<int32_t> GPR_CLAMP(
+    uint32_t frame_size = static_cast<uint32_t> GPR_CLAMP(
```

Comment:
why do we want this to be unsigned? The first argument to GPR_MAX is cast to int32_t (in C style, which we should probably update too), which is signed.